packages/terra-application-utility/src/utility/_UtilityMenuDivider.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './_UtilityMenuDivider.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * If the divider shall be placed before the footer
+   */
+  isFooter: PropTypes.bool,
+};
+
+const UtilityMenuDivider = ({ isFooter, ...customProps }) => {
+  const dividerClassNames = cx([
+    'divider',
+    { 'is-header': !isFooter },
+    { 'is-footer': isFooter },
```

Comment:
If you change this to isTop, I'd change `is-header` to `isBottom`

Reply:
A little confused here - are you saying to refactor the selectors into:
 ``` javascript
{ is-top : isTop }
{ is-bottom: !isTop }
```
?

Reply:
Yep